taylor-fatima.js

```diff
@@ -0,0 +1,143 @@
+
+//Part 1
+//#1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+
+function calculateCube(num) {
+    console.log(Math.pow(num, 3))
+  }
+  calculateCube(2)
+  
+  //#2
+  //Write a function isAVowel that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+  
+  function isAVowel(letter) {
+    if (letter === 'a') {
+      return true
+    } else if (letter === 'e'){
+      return true
+    } else if (letter === 'i') {
+      return true
+    } else if (letter === 'o') {
+      return true 
+    } else if (letter === 'u') {
+      return true
+    } else {
+      return false
+    }
+  }
+  console.log(isAVowel('a'))
```

Comment:
Nice work here! one thing that might throw a wrench in you logic is that your function doesn't account for an uppercase letter.